.changeset/eighty-bears-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': major
+---
+
+Escape toast HTML content that is not wrapped in "-pharos-" tag
```

Comment:
I wonder if theres any way we could feel comfortable making this a patch release? It might not be possible but that will lower the barrier of entry to release the update

Reply:
I can change it. I was erroring on the side of caution since this could be a breaking change.

Reply:
I could see us having a brief discussion with the group. I can definitely see where you were coming from as things could break but is it OK that they break if it fixes a semi-vulnerability?